### ISSUE TITLE
[SEQ-1740] Toolbar uses glassy treatment, time ruler does not. 

### ISSUE THREAD

**Issue by ParkerGibbons** [sequencefilm/docs-and-issues#11]
Noticed this UI inconsistency and wanted to call it out. Would be cool for both to have a glassy treatment!

Toolbelt: glassy treatment

![](https://uploads.linear.app/11eb39a2-0318-45d2-a3dc-efb1c6029d82/f9ec85f2-2c21-4651-93bb-d25cdbf17bd2/61aef205-f66c-437c-be37-6569f852ba11)

Time Ruler: no glassy treatment

![](https://uploads.linear.app/11eb39a2-0318-45d2-a3dc-efb1c6029d82/873d98e8-6df0-4150-8761-d0bfbaf7eac5/e1e9c628-1516-40e1-9fab-aa2daa170a09)

<sub>From [SyncLinear.com](https://synclinear.com) | [SEQ-1740](https://linear.app/sequencefilm/issue/SEQ-1740/toolbar-uses-glassy-treatment-time-ruler-does-not)</sub>

**Comment by lmcgartland** [comment]
Yeah but it looks cool 😎 We actually have this in figma, it's a darker glass so the ticks/text are still readable (not sure how well it meets the standards). @pgibbons can you put in the figma link?
